src/models/rule_store.rs

```diff
@@ -34,14 +34,15 @@ use crate::{
 };
 
 use super::{
+  language::PiranhaLanguage,
   outgoing_edges::Edges,
   rule::{InstantiatedRule, Rules},
 };
 
 pub(crate) static GLOBAL: &str = "Global";
 pub(crate) static PARENT: &str = "Parent";
 /// This maintains the state for Piranha.
-#[derive(Debug, Getters)]
+#[derive(Debug, Getters, Default)]
```

Comment:
Just to make sure I get this right: Before this change we needed some custom logic on our `Default` implementation for `RuleStore` (namely the `PiranhaArgumentsBuilder::default().build()` line) and thus we couldn't use `#[derive(Default)]` here, but now we can? 

Is this correct? I see `PiranhaArguments` already has `#[derive(Default)]` on master. What about the main `Piranha` struct? No need to change that since there isn't a `Default` implementation?

Reply:
Yes. You are correct about why we do not require to `impl Default for RuleStore`  anymore because we do not need `PiranhaArgumentsBuilder::default().build()` (since we removed `PiranhaArguments` from `RuleStore`).

That's an awesome catch. While `Piranha` doesnt need `PiranhaArguments`'s `Default` trait, but the function `read_toml` relies on the `PiranhaArguments::default()` when the `piranha_arguments.toml` file is not provided, but we expect one.  Its more like a error scenario. ( I dont see that happening anymore), since we can fully compose `piranha_aguments.toml` from commandline/api now. 

But for consistency, and future bug proofing I have removed the `#[derive(Default)]` from `PiranhaArguments`  and implemented the `Default` for it by hand.